buildSrc/build.gradle.kts

```diff
@@ -1,3 +1,11 @@
+
+import org.gradle.api.JavaVersion
```

Comment:
:x: Most of these imports are implicit and not needed

Reply:
And the imports should be moved below the license header

Reply:
How do I tell that to IDEA?

Reply:
IOW: I had "organize imports" switched on and that is what I ended up with.

Reply:
If using project import then the implicit imports should be handled by IntelliJ.

I don't use the "on the fly imports" feature and, using e.g. `JavaVersion` doesn't add any import statement as it is already available via implicit imports. When triggering "organize imports" it doesn't add it either.

![image](https://user-images.githubusercontent.com/132773/34299156-bd85e724-e721-11e7-8e08-6e7eeca3a1e4.png)

What are your settings?

About putting the imports below the licence header, I don't know how to tell that to IntelliJ.

Reply:
"Optimize imports on the fly" seems to be the culprit.

Reply:
I fixed the import statements manually, please have another look.

I think Jetbrains should fix the problems with "Optimize imports on the fly" and Kotlin build scripts. I had a hard time importing the classes anyway, the popup didn't show the corresponding classes (Alt-Return). There may also be some problems there.

Reply:
@wolfs I just opened https://youtrack.jetbrains.com/issue/KT-21981, please upvote!